# HG changeset patch
# User Markus Mützel <markus.muetzel@gmx.de>
# Date 1786936982 25200
#      Sun Aug 16 20:23:02 2026 -0700
# Node ID 9eb89d53a0433322ea8e08c12e2ba4747a948473
# Parent  4b0d61c3377b8d26d5ec6c093625dd749d91e0bc
Don't use short paths (bug #59923).

diff -r 4b0d61c3377b -r 9eb89d53a043 octave-firsttime.vbs
--- a/octave-firsttime.vbs	Sat Aug 15 08:52:26 2026 -0400
+++ b/octave-firsttime.vbs	Sun Aug 16 20:23:02 2026 -0700
@@ -33,11 +33,6 @@ If objFSO.FileExists(OctavePath & "\ming
   OctavePath = OctavePath & "\mingw32" 
 End If
 
-' get path as a 8.3 path
-Set fo = fso.GetFolder(OctavePath)
-OctavePath = fo.ShortPath
-Set fo = Nothing
-
 ' set up path to ensure octave bin comes first
 Set wshSystemEnv = wshShell.Environment( "PROCESS" )
 if OctavePath <> MSysPath Then
@@ -54,9 +49,6 @@ wshSystemEnv("GS") = "gs.exe"
 
 If wshShell.ExpandEnvironmentStrings("%HOME%") = "%HOME%" Then
   Home = wshSystemEnv("USERPROFILE")
-  Set fo = fso.GetFolder(Home)
-  wshSystemEnv("HOME") = fo.ShortPath
-  Set fo = Nothing
 End If
 
 ' set Qt plugin directory and path 
diff -r 4b0d61c3377b -r 9eb89d53a043 octave.vbs
--- a/octave.vbs	Sat Aug 15 08:52:26 2026 -0400
+++ b/octave.vbs	Sun Aug 16 20:23:02 2026 -0700
@@ -32,11 +32,6 @@ ElseIf objFSO.FileExists(OctavePath & "\
   OctavePath = OctavePath & "\mingw32" 
 End If
 
-' get path as a 8.3 path
-Set fo = fso.GetFolder(OctavePath)
-OctavePath = fo.ShortPath
-Set fo = Nothing
-
 ' set up path to ensure octave bin comes first
 Set wshSystemEnv = wshShell.Environment( "PROCESS" )
 if OctavePath <> MSysPath Then
@@ -54,9 +49,6 @@ wshSystemEnv("GS") = "gs.exe"
 
 If wshShell.ExpandEnvironmentStrings("%HOME%") = "%HOME%" Then
   Home = wshSystemEnv("USERPROFILE")
-  Set fo = fso.GetFolder(Home)
-  wshSystemEnv("HOME") = fo.ShortPath
-  Set fo = Nothing
 End If
 
 ' set Qt plugin directory and path 
diff -r 4b0d61c3377b -r 9eb89d53a043 post-install.bat
--- a/post-install.bat	Sat Aug 15 08:52:26 2026 -0400
+++ b/post-install.bat	Sun Aug 16 20:23:02 2026 -0700
@@ -1,8 +1,6 @@
 @echo off
 Rem   Run on initial install /update
 set ROOT_DIR=%~dp0
-Rem Convert to 8.3 format so we don't have to worry about spaces.
-for %%I in ("%ROOT_DIR%") do set ROOT_DIR=%%~sI
 
 set MSYSDIR=%ROOT_DIR%
 set MSYSTEM=MSYS
